### DISCUSSION TITLE
change author

### DISCUSSION THREAD

**Discussion by ysahih** [community/community#113721] [New to GitHub] [answered]
how can i change the author of a commit ?

**Comment by saifeddineelhanoune** [comment, marked as answer]
To change the author of the last commit, use git commit --amend --author="New Author Name <new-email@example.com>"

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
